crates/cli/src/main.rs

```diff
@@ -95,14 +101,17 @@ fn main() -> Result<()> {
         paths.push(canonicalized.to_string(|path| path.display().to_string()))
     }
 
+    // {"server_url": "http://localhost:8080"}
```

Comment:
Are these comments leftover from development?

Reply:
From explaining stuff to @SomeoneToIgnore, thanks (as usual) for catching